lib/tasks/ingest.rake

```diff
@@ -423,7 +423,7 @@ namespace :ingest do
         school.updated_at = Time.zone.now
         school.save
 
-        break if Rails.env.test? && index == 2 # import_schools_test.rb assumes that the CSV only has one record
+        break if Rails.env.test? # import_schools_test.rb only needs one record
```

Comment:
Why did you let import_schools_test.rb test one record vs opposed to all of them?

Reply:
My goal was to test that the rake task saves a record properly rather than the quality of the data.  Testing one can save some time.  Sound reasonable?